frontend/src/pages/budgetLines/PreviewTable.jsx

```diff
@@ -20,7 +20,9 @@ export const PreviewTable = ({ handleDeleteBudgetLine = () => {}, readOnly = fal
         .sort((a, b) => Date.parse(a.created_on) - Date.parse(b.created_on))
         .reverse();
 
-    let loggedInUser = useSelector((state) => state.auth.activeUser.full_name);
+    let loggedInUser = useSelector((state) =>
+        state.auth?.activeUser ? state.auth.activeUser.full_name : "Not logged in"
```

Comment:
Maybe we can make a reusable helper function to display the user name. 1st try `full_name`, if not available then use `email`, and last option something like `unknown user`.